### ISSUE TITLE
[BUG] FPS drops at Nefarian P3

### ISSUE THREAD

**Issue by heyiamsven** [dfherr/ThreatClassic2#33]
### What steps will reproduce the problem?  
1.  As the title says
2.  People are using TC2 or Details Tiny Threat

### What version of ThreatClassic2 are you using? (Stating 'latest' is not useful)
latest and 5-6 of our member had outdated versions of TC2 (maybe 1-2 revisions back)

### Do you have an error log of what happened?
No

### Any additional information? (example: WoW language if not English) (Attach screenshots here if available, do not link externally)
Other users seem to have the same problem according to this reddit
https://www.reddit.com/r/classicwow/comments/f5nh60/big_frame_drop_during_nefarian_phase_3/

Any idea why this happens or is it maybe caused by Details! tiny threat syncing with TC2 users through LibThreatClassic2?

**Comment by heyiamsven** [comment]
Blacklisted now Skeletons, Infernals, Totems from Nef Encounter, lets see tomorrow if this will minimize CPU usage. Will commit if this works!

**Comment by dev7355608** [comment]
I already opened a PR for the Corrupted Infernals; they definitely cause lag. The Bone Constructs and the Corrupted Totems should be blacklisted as well. Am I missing any totems on this list?
```
14663 Corrupted Stoneskin Totem
14664 Corrupted Healing Stream Totem
14666 Corrupted Windfury Totem
14667 Corrupted Fire Nova Totem
```

**Comment by heyiamsven** [comment]
> I already opened a PR for the Corrupted Infernals; they definitely cause lag. The Bone Constructs and the Corrupted Totems should be blacklisted as well. Am I missing any totems on this list?
> 
> ```
> 14663 Corrupted Stoneskin Totem
> 14664 Corrupted Healing Stream Totem
> 14666 Corrupted Windfury Totem
> 14667 Corrupted Fire Nova Totem
> ```

Here you go, everything that could cause a lot of Threat Calculations:
	[14605] = true,		-- Bone Construct
	[14668] = true,		-- Corrupted Infernal
	[23423] = true,		-- Corrupted Winfury Totem
	[23419] = true,		-- Corrupted Fire Nova Totem
	[23422] = true,		-- Corrupted Healing Stream Totem
	[23420] = true,		-- Corrupted Stoneskin Totem

**Comment by dev7355608** [comment]
Thanks! I added them all to dfherr/LibThreatClassic2#20.

**Comment by dfherr** [comment]
hopefully fixed with v2.12
